src/class-wpml-gutenberg-integration-factory.php

```diff
@@ -41,8 +42,13 @@ public function create() {
 			$integrations->add(
 				new WPML\PB\Gutenberg\Reusable_Blocks_Admin_Integration(
 					new WPML\PB\Gutenberg\Reusable_Blocks_Batch_Handler(
-						new WPML\PB\Gutenberg\Reusable_Blocks(),
+						$reusable_blocks,
 						$reusable_blocks_translation
+					),
+					new WPML\PB\Gutenberg\Reusable_Blocks_Basket_Handler(
+						$reusable_blocks,
+						$reusable_blocks_translation,
+						new \WPML_Translation_Basket( $wpdb )
```

Comment:
@strategio We could use the DI (see what I did here? ;) ) helper for creating the instance of `\WPML_Translation_Basket`.

That would help to test this code as well.

Reply:
Done @andreasciamanna https://github.com/OnTheGoSystems/wpml-page-builders-gutenberg/pull/37/commits/92d5d7397ad5a713d218e9e6f2e880a6671698f6